Jenkinsfile

```diff
@@ -17,19 +22,22 @@ pipeline {
       agent {
         dockerfile { dir 'webpagetest-api' }
       }
-      environment {
-        WEB_PAGE_TEST = credentials('WEB_PAGE_TEST')
-        WEBPAGETEST_SERVER = "https://${WEB_PAGE_TEST}@wpt-api.stage.mozaws.net/"
-        PAGE_URL = "https://latest.dev.lcip.org/?service=sync&entrypoint=firstrun&context=fx_desktop_v3"
-      }
       steps {
-          sh '/usr/src/app/bin/webpagetest test "${PAGE_URL}" -l "us-east-1:Firefox" -r 9 --first --poll --reporter json > fxa-homepage.json'
+        sh '/usr/src/app/bin/webpagetest test "${PAGE_URL}" -l "us-east-1:Firefox" -r 9 --first --poll --reporter json > fxa-homepage.json'
       }
       post {
         always {
           archiveArtifacts 'fxa-homepage.json'
```

Comment:
If your `jq` stage is modifying this file, then you probably want to move the archiving of it to after your final stage.

Reply:
Oh, thanks, good point; I had thought about that before implementing the ```jq``` portion, but then got swept away in implementing ```jq``` itself, and forgot to revisit this.  I'll rework this in this patch.

Prior to reading this comment, I also just filed https://github.com/mozilla/wpt-api/issues/9.

Reply:
Actually, I'd like to keep ```fxa-homepage.json``` and ```stats.json``` separate, for now, so I can work with them separately - that might, of course, change, down the road, as I get traction/adoption.